# Goal
std_msgs/Header header
geometry_msgs/Pose pose
---
# Define the result
bool success
---
# Define a feedback message
float32 goal_distance